fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java

```diff
@@ -78,11 +95,14 @@ public boolean visit(String elementName, int elementIndex, Visitable visitable)
     /*====================
      * Primitive Types   *
      ====================*/
-    
+
     @Override
     public boolean visit(java.lang.String elementName, int elementIndex, com.ibm.fhir.model.type.Boolean _boolean) {
         Parameter p = new Parameter();
-        p.setName(name);
+        if (!TOKEN.equals(searchParamType)) {
+            throw invalidComboException(searchParamType, _boolean);
+        }
+        p.setName(searchParamCode);
         p.setValueSystem("http://hl7.org/fhir/special-values");
```

Comment:
should this be: http://hl7.org/fhir/ValueSet/special-values

Reply:
https://www.hl7.org/fhir/search.html#token lists the exact value I use here:
> The implicit system for boolean values is http://hl7.org/fhir/special-values but this is never actually used

Its possible that they just never updated that with the ValueSet URL you sent (which definitely has true and false).  However, in that case the correct value to use would be `http://terminology.hl7.org/CodeSystem/special-values` and not the one you suggested here.

Reply:
I opened https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&tracker_item_id=25124 and will update the PR to use this new value instead

Reply:
```suggestion
        p.setValueSystem("http://terminology.hl7.org/CodeSystem/special-values");
```